packages/gen-ai-hub/README.md

```diff
@@ -0,0 +1,32 @@
+# @sap-ai-sdk/gen-ai-hub
+
+This package incorporates generative AI into your AI activities in SAP AI Core and SAP AI Launchpad.
+
+### Installation
+
+```
+$ npm install @sap-ai-sdk/gen-ai-hub
+```
+
+## Pre-requisites
```

Comment:
[q] Same as above, can we link it to root and link?

Reply:
This is shown on the npmjs page of the package, I wouldn't link it to the root README and force the user to navigate to the monorepo.